HW1/P4/P4_bfs.py

```diff
@@ -0,0 +1,31 @@
+import time
+
+num_of_partitions = 2
+
+def copartitioned(RDD1, RDD2):
+    "check if two RDDs are copartitioned"
+    return RDD1.partitioner == RDD2.partitioner
+
+def SSBFS( source, graph, accum ): 
+    graph = graph.map( lambda ( x, y ): ( x, ( -1, y ) ) if x != source else ( x, ( 0, y ) ), preservesPartitioning = True )
+    graph.cache()
+    i = 0
```

Comment:
nit: you have 0 hardcoded 2x, you could have separated it out into an initial_distance variable or somesuch